modules/cnf-image-based-upgrade-guidelines.adoc

```diff
@@ -0,0 +1,197 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-guide_{context}"]
+= Guidelines for the image-based upgrade
+
+For a successful image-based upgrade, your deployments need to consider certain requirements.
+
+There are different deployment methods in which you can perform the image-based upgrade:
+
+{ztp}:: You utilize the {ztp-first} to deploy and configure your clusters.
+Non-GitOps:: You only use {rh-rhacm-first} to deploy and configure your clusters.
+
+[id="ztp-image-based-upgrade-cluster-validated-software_{context}"]
+== Minimum software version of components
+
+Depending on your deployment method, the image-based upgrade requires the following minimum software versions.
+
+.Minimum software version of components
+[cols=3*, width="80%", options="header"]
+|====
+|Component
+|Software version
+|Required
+
+|{lcao}
+|4.16
+|Yes
+
+|OADP Operator
+|1.3.1
+|Yes
+
+|Managed cluster version
+|4.14.13
+|Yes
+
+|Hub cluster version
+|4.16
+|Yes
+
+|{rh-rhacm}
+|2.10.2
+|Yes
+
+|{ztp} plugin
+|4.16
+|Only for {ztp} deployment method
+
+|{gitops-title}
+|TBD
+|Only for {ztp} deployment method
+
+|{cgu-operator-first}
+|4.16
+|Only for {ztp} deployment method
+
+|Local Storage Operator ^[1]^
+|TBD
+|Yes
+
+|{lvms-first} ^[1]^
+|4.14.2
+|Yes
+|====
+. The persistent storage must be provided by either the {lvms} or the Local Storage Operator, not both.
+
+[id="ztp-image-based-upgrade-hub-cluster-guide_{context}"]
+== Hub cluster guidelines
+
+If you are using {rh-rhacm-first}, your hub cluster needs to meet the following conditions:
+
+* To avoid including any {rh-rhacm} resources in your seed image, you need to disable all optional {rh-rhacm} add-ons for the seed image generation.
```

Comment:
```suggestion
* To avoid including any {rh-rhacm} resources in your seed image, you need to disable all optional {rh-rhacm} add-ons before initiating seed image generation or before generating seed image.
```

Reply:
I feel this could lead to confusion but maybe I'm just not seeing the difference between "before initiating seed image generation" vs "before generating seed image". Why isn't only one of them enough here?

Reply:
yes, I have given 2 options , we can implement either of them as per our documentation guidelines.

Reply:
Okay, going with "before generating seed image" then